src/localization.directives.js

```diff
@@ -63,28 +65,40 @@ angular.module('ngLocalize')
 
                 locale.ready(langFiles).then(function () {
                     var value = '';
-
                     for(var key in values) {
                         exp = values[key];
                         value = locale.getString(exp);
                         if (lastValues[key] !== value) {
-                            attrs.$set(attrs.$normalize(key), lastValues[key] = value);
+                            lastValues[key] = value;
+                            setAttr(attrs, key, value);
                         }
                     }
                 });
-            }
+            };
+        }
+        return {
+            priority: 1000,
+            compile: function (elem, attrs) {
+                angular.forEach($rootScope.$eval(attrs.i18nAttr), function (value, key) {
+                    setAttr(attrs, key, value || '...');
```

Comment:
an error will be thrown with and empty aria-label. Do you have any thoughts on how best to handle? We just to temporarily populate the aria-labels so that any warning in the console is not a false positive.